airflow/providers/postgres/hooks/postgres.py

```diff
@@ -191,7 +203,7 @@ def get_iam_token(self, conn):
         return login, token, port
 
     @staticmethod
-    def _generate_insert_sql(table, values, target_fields, replace, **kwargs):
+    def _generate_insert_sql(table: str, values: Tuple[object, ...], target_fields: Iterable[str], replace: bool, **kwargs) -> str:
```

Comment:
```suggestion
    def _generate_insert_sql(table: str, values: Tuple[str, ...], target_fields: Iterable[str], replace: bool, **kwargs) -> str:
```
According to `DbApiHook._serialize_cell`

Reply:
@turbaszek  I will fix that right now :+1: